tutorial INBO citation style (#212)
inbo/tutorials: content/tutorials/style_inbo_bibliography/index.Rmd

bibliography: "../../articles/reproducible_research.bib"
output: 
    md_document:
        preserve_yaml: true
        variant: gfm
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# The INBO citation style

Most journals, publishers and companies use a specified format for citations and bibliographies.
The Research Institute for Nature and Forest (INBO) is no exception.

All bibliography managers provide ways to automate the formatting of citations and bibliographies which greatly facilitates scientific publishing.
Depending on the bibliographic manager software, the style formatting rules are stored in specific files.
In this tutorial you will learn how to use the INBO citation style in EndNote, Mendeley and Zotero bibliography managers.

EndNote uses a `*.ens` file to store the style formatting rules, whereas Mendeley and Zotero use the open source `*.csl` file (Citation Style Language)[https://citationstyles.org/].

The latest official CSL file of the INBO citation style is available at the CSL style repository (also transferred automatically to the Zotero style repository).

> See (finding-and-installing-styles)[https://citationstyles.org/authors/#/finding-and-installing-styles] for more information.

Reviewer: Once the brackets are corrected, this link will render as:

> See [finding-and-installing-styles](https://citationstyles.org/authors/#/finding-and-installing-styles) for more information.

I'm not sure if that was your intention. (It might be)

Here are direct links to the CSL file:

- (file in the CSL style repository)[https://github.com/citation-style-language/styles/blob/master/research-institute-for-nature-and-forest.csl]
- (file in the Zotero style repository)[https://www.zotero.org/styles/research-institute-for-nature-and-forest]

> The link to the Zotero style repository lets you directly download the style file.

Author: Good point. Will change it.

In case older versions are needed, they can be obtained from successively numbered releases from our (forked styles repo)[https://github.com/inbo/styles/releases].

The EndNote `*.ens` file can be downloaded (here)[https://sites.google.com/a/inbo.be/intranet/ondersteuning/bibliotheek/endnote-informatie/endnotehuisstijl] (only accessible for INBO employees).

Curious what the formatted INBO citation style and bibliography looks like for various document types?
Take a look at these links:

- [PDF based on CSL file](https://github.com/inbo/inbomd_examples/tree/master/source/citation_style/en/citation_style_csl.pdf)
- [HTML based on CSL file](https://github.com/inbo/inbomd_examples/tree/master/source/citation_style/en/index.html)

# Use in bibliography managers

## Zotero

1. Open Zotero
1. Go to `Edit > Preferences > Cite` and click on "Get additional styles..."

    ![](zotero_styles.png)

1. In the search field in the popup window that opens type "Research Institute for Nature and Forest" and click on the corresponding link

> There are good online tutorials that explain how to use Zotero in combination with [google docs](https://www.zotero.org/support/google_docs) and [MS Word](https://www.zotero.org/support/word_processor_plugin_usage). 

Reviewer: ```suggestion
There are good online tutorials that explain how to use Zotero in combination with [google docs](https://www.zotero.org/support/google_docs), [MS Word](https://www.zotero.org/support/word_processor_plugin_usage) and [LibreOffice](https://www.zotero.org/support/libreoffice_writer_plugin_usage). 
```

I saw you mentioned LibreOffice for Mendeley, so adding it here.

Depending on the bibliographic manager software, the style formatting rules are stored in specific files.
In this tutorial you will learn how to use the INBO citation style in EndNote, Mendeley and Zotero bibliography managers.

> EndNote uses a `*.ens` file to store the style formatting rules, whereas Mendeley and Zotero use the open standard `*.csl` [Citation Style Language](https://citationstyles.org/).

Reviewer: ```suggestion
EndNote uses a `*.ens` file to store the style formatting rules, whereas Mendeley and Zotero use the open standard `*.csl` ([Citation Style Language](https://citationstyles.org/)).
```
I think it's best in another pair of brackets here :wink:

EndNote, Zotero and Mendeley have recommended ways of finding and installing these files and will always point to the latest version of the style files when properly installed.
This is explained in [Use in bibliography managers].

> The latest official CSL file of the INBO citation style is available at the CSL style repository (also transferred automatically to the Zotero style repository) and can be found [here](https://github.com/citation-style-language/styles/blob/master/research-institute-for-nature-and-forest.csl).

Reviewer: ```suggestion
In case you want to manually download the latest official CSL file of the INBO citation style (not the recommended way; better use your bibliography manager): it is [available](https://github.com/citation-style-language/styles/blob/master/research-institute-for-nature-and-forest.csl) at the CSL style repository.
```

In case older versions are needed, they can be obtained from successively numbered releases from our [forked styles repo](https://github.com/inbo/styles/releases).

> To use older versions, you will need to manually download and add the file to your reference manager.

Reviewer: ```suggestion
To use older versions, you will need to manually download and add the file to your bibliography manager.
```

The EndNote `*.ens` file can be downloaded [here](https://sites.google.com/a/inbo.be/intranet/ondersteuning/bibliotheek/endnote-informatie/endnotehuisstijl) (only accessible for INBO employees).

> Curious what the formatted INBO citation style and bibliography looks like for various document types?

Reviewer: ```suggestion
Curious how the formatted INBO citation style and bibliography look like for various document types?
```
